src/main/java/org/apache/commons/imaging/ImageParser.java

```diff
@@ -898,7 +892,7 @@ public boolean canAcceptType(final ImageFormat type) {
      * @param file An valid file reference.
      * @return If the parser can accept the format, true; otherwise, false.
      */
-    protected final boolean canAcceptExtension(final File file) {
+    public boolean canAcceptExtension(final File file) {
```

Comment:
Made it public because I created methods in the new internal `Util` class. Similar to the already public `canAcceptType(ImageFormat)` of this class.